sdk/communication/azure-communication-callingserver/src/main/java/com/azure/communication/callingserver/models/CallingServerErrorException.java

```diff
@@ -55,13 +52,15 @@ public CallingServerErrorException() {
                 @Override
                 public CallingServerErrorException create(HttpResponseException internalHeaders) {
                     CallingServerError error = null;
-                    if (internalHeaders.getValue() != null) {
-                        ObjectMapper objectMapper = new ObjectMapper();
-                        objectMapper.disable(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES);
-                        CommunicationErrorResponse communicationErrorResponse = objectMapper.convertValue(
-                            internalHeaders.getValue(), CommunicationErrorResponse.class);
-                        error = convert(communicationErrorResponse.getError());
-                    }
+// TODO (anu) Sync with Alan
+//
+//                    if (internalHeaders.getValue() != null) {
+//                        ObjectMapper objectMapper = new ObjectMapper();
+//                        objectMapper.disable(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES);
+//                        CommunicationErrorResponse communicationErrorResponse = objectMapper.convertValue(
+//                            internalHeaders.getValue(), CommunicationErrorResponse.class);
+//                        error = convert(communicationErrorResponse.getError());
```

Comment:
This is something to follow up with Alan. i.e., how to deser the Object `internalHeaders.getValue()`